src/test/modelinstance_test.cpp

```diff
@@ -287,6 +287,86 @@ TEST_F(TestLoadModel, UnSuccessfulLoadWhenNireqTooHigh) {
     EXPECT_EQ(ovms::ModelVersionState::LOADING, modelInstance.getStatus().getState()) << modelInstance.getStatus().getStateString();
 }
 
+class TestLoadModelWithMapping : public TestLoadModel {
+protected:
+    void SetUp() override {
+        TestLoadModel::SetUp();
+        config = DUMMY_MODEL_CONFIG;
+        ovms::mapping_config_t mappingOutputs{{"a", "output"}};
+        ovms::mapping_config_t mappingInputs{{"b", "input"}};
+        config.setMappingInputs(mappingInputs);
+        config.setMappingOutputs(mappingOutputs);
+
+        ovms::mapping_config_t realMappingOutputs{{"output", "a"}};
+        ovms::mapping_config_t realMappingInputs{{"input", "b"}};
+        config.setRealMappingInputs(realMappingInputs);
+        config.setRealMappingOutputs(realMappingOutputs);
+    }
+    ovms::ModelConfig config;
+    ovms::shapes_map_t shapeMap;
+    ovms::layouts_map_t layouts;
+};
+
+TEST_F(TestLoadModelWithMapping, SuccessfulLoad) {
+    ovms::ModelInstance modelInstance("UNUSED_NAME", UNUSED_MODEL_VERSION, *ieCore);
+
+    ovms::ShapeInfo inputShape{ovms::FIXED, {1, 20}};
+    shapeMap["input"] = inputShape;
+    config.setShapes(shapeMap);
+
+    layouts["input"] = "LAYOUT_INPUT";
```

Comment:
Should it really work? `LAYOUT_INPUT` is not supported layout, only `NHWC` and `NCHW`